### PR TITLE
[AKS] SL-1855: add history tab to hearing details screen

### PR DESCRIPTION
**Pull request by r-kruk-kainos**

### CODE REVIEW

File: src/app/core/pipes/date-format.pipe.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+import * as moment from 'moment';
+
+@Pipe({ name: 'appDateTimeToDate' })
+export class DateFormatPipe implements PipeTransform {
+    transform(dateTime: moment.Moment) {
+        return moment(dateTime).format();
```
Comment (radoslawlandowskikainos) [inline src/app/core/pipes/date-format.pipe.ts]: I've created a simmilar pipe for my pr. Let's extend it and make it parametrized:

    transform(dateTime: moment.Moment, timeFormat: string) {
        return timeFormat !== undefined ? dateTime.format(timeFormat) : dateTime.format();
    }

you can then call it like so: {{date | momentFormat:'dd/mm/yyy'}}

Comment (radoslawlandowskikainos) [reply src/app/core/pipes/date-format.pipe.ts]: And call it generically: MomentFormatPipe

File: src/app/features/activityLog/models/activity-log.model.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import * as moment from 'moment';
+
+export interface ActivityLog {
+    activityStatus: string,
+    description: string,
+    createdBy: string,
+    createdAt: moment.Moment;
```
Comment (radoslawlandowskikainos) [inline src/app/features/activityLog/models/activity-log.model.ts]: Where do you transform the string from response to moment object?

File: src/app/features/activityLog/services/activity-log.service.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+import { ActivityLog } from '../models/activity-log.model';
+
+@Injectable()
+export class ActivityLogService {
+    constructor(private readonly http: HttpClient, private readonly config: AppConfig) {
+    }
+
+    getActivitiesForEntity(entityId: string): Observable<ActivityLog[]> {
+        return this.http
+            .get<ActivityLog[]>(`${this.getUrl()}/${entityId}`)
```
Comment (radoslawlandowskikainos) [inline src/app/features/activityLog/services/activity-log.service.ts]: Shouldn't you transform to moment object here?

File: src/app/features/activityLog/components/activities-log.component.html [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    <mat-list-item *ngFor="let activityLog of activities">
+        <div matLine fxLayout="row">
+            <div fxFlex>
+                <strong>{{activityLog.activityStatus}}</strong>
+            </div>
+            <div fxFlex>
+                {{activityLog.createdBy}}
+            </div>
+
+            <div fxFlex>
```
Comment (zielinskikamil) [inline src/app/features/activityLog/components/activities-log.component.html]: formatting + question, shouldn't we have here some max boundaries?
like status 50% of available space or createdBy aligned to the right edge?

File: src/app/hearing/components/view-hearing/view-hearing.component.ts [hunk 5]
```diff
@@ -105,6 +111,6 @@ export class ViewHearingComponent implements OnInit {
     }
 
     private fetchHearing() {
-        this.hearingService.getById(this.hearingId);
+        this.hearingService.getById(this.hearingId).subscribe();
```
Comment (zielinskikamil) [inline src/app/hearing/components/view-hearing/view-hearing.component.ts]: try to refactor it to not return observable, but just void.

Discussed offline

**Pull request merged**